nncf/openvino/graph/node_utils.py

```diff
@@ -468,3 +472,23 @@ def get_weighted_layer_attributes(
         }
         return LinearLayerAttributes(**kwargs)
     return GenericWeightedLayerAttributes(weight_requires_grad=False, weight_shape=attrs.get("shape", None))
+
+
+def get_activation_channel_axis(node: NNCFNode, port_id: int, input_shape: Tuple[int]) -> int:
+    """
+    Returns axis number of the activation tensor which correspond to it channel.
+
+    :param node: NNCFNode instance.
+    :param port_id: Port ID for input.
+    :param input_shape: Shape of the input.
+    :return: Channel axis number.
+    """
+    # In case of the OpenVINO, [N, C, ..] layout applicable for most quantizable layers.
+    channel_axis = 1
+
+    # But the MatMul layers may transpose inputs internally.
+    if node.metatype == OVMatMulMetatype:
+        activations_layout = get_linear_activations_layout_from_node(node, port_id, input_shape)
+        channel_axis = activations_layout.index(OVLayoutElem.C_IN)
+
+    return channel_axis
```

Comment:
My 5 cents
```suggestion
    # In case of the OpenVINO, [N, C, ..] layout applicable for most quantizable layers.
    if node.metatype != OVMatMulMetatype:
        return 1

    # But the MatMul layers may transpose inputs internally.
    activations_layout = get_linear_activations_layout_from_node(node, port_id, input_shape)
    return activations_layout.index(OVLayoutElem.C_IN)
```

Reply:
Why this is better? The suggested code contains an unnamed `return 1` that is not a transparent solution here.